handlers/search.go

```diff
@@ -237,22 +244,18 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 		if utils.IsValidEnsDomain(search) {
 			ensData, _ = GetEnsDomain(search)
 			if len(ensData.Address) > 0 {
-				search = strings.Replace(ensData.Address, "0x", "", -1)
+				lowerStrippedSearch = strings.Replace(ensData.Address, "0x", "", -1)
 			}
 		}
 		if !searchLikeRE.MatchString(search) {
 			break
 		}
-		if len(search)%2 != 0 { // pad with 0 if uneven
-			search = search + "0"
-		}
```

Comment:
**Question:** Why has this been removed?

Reply:
Because it makes no sense now that we use text search in the query. It just leads to not finding stuff with uneven input.